docs/guide/acquiring_data/fido.rst

```diff
@@ -180,131 +167,347 @@ client for results, in this case VSO.::
+    2011-01-01 00:09:48.000        AIA intensity   211.0 .. 211.0
+    2011-01-01 00:09:50.000        AIA intensity     94.0 .. 94.0
+    2011-01-01 00:09:51.000        AIA intensity   335.0 .. 335.0
+    2011-01-01 00:09:53.000        AIA intensity 1600.0 .. 1600.0
+    2011-01-01 00:09:55.000        AIA intensity   193.0 .. 193.0
+    2011-01-01 00:09:56.000        AIA intensity   304.0 .. 304.0
+    2011-01-01 00:09:57.000        AIA intensity   131.0 .. 131.0
+    2011-01-01 00:10:00.000        AIA intensity   171.0 .. 171.0
+    2011-01-01 00:10:00.000        AIA intensity   211.0 .. 211.0
+    Length = 402 rows
+    <BLANKLINE>
+    42 Results from the VSOClient:
+           Start Time       Instrument        Physobs         Wavelength [2]
+                                                                 Angstrom
+    ----------------------- ---------- --------------------- ----------------
+    2011-01-01 00:00:00.000        HMI VECTOR_MAGNETIC_FIELD 6173.0 .. 6173.0
+    2011-01-01 00:00:00.000        HMI VECTOR_MAGNETIC_FIELD 6173.0 .. 6173.0
+    2011-01-01 00:00:00.000        HMI    LOS_magnetic_field 6173.0 .. 6173.0
+    2011-01-01 00:00:25.000        HMI             intensity 6173.0 .. 6174.0
+    2011-01-01 00:00:25.000        HMI    LOS_magnetic_field 6173.0 .. 6174.0
+    2011-01-01 00:00:25.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    2011-01-01 00:01:10.000        HMI             intensity 6173.0 .. 6174.0
+    2011-01-01 00:01:10.000        HMI    LOS_magnetic_field 6173.0 .. 6174.0
+    2011-01-01 00:01:10.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    2011-01-01 00:01:55.000        HMI             intensity 6173.0 .. 6174.0
+                        ...        ...                   ...              ...
+    2011-01-01 00:07:10.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    2011-01-01 00:07:55.000        HMI             intensity 6173.0 .. 6174.0
+    2011-01-01 00:07:55.000        HMI    LOS_magnetic_field 6173.0 .. 6174.0
+    2011-01-01 00:07:55.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    2011-01-01 00:08:40.000        HMI             intensity 6173.0 .. 6174.0
+    2011-01-01 00:08:40.000        HMI    LOS_magnetic_field 6173.0 .. 6174.0
+    2011-01-01 00:08:40.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    2011-01-01 00:09:25.000        HMI             intensity 6173.0 .. 6174.0
+    2011-01-01 00:09:25.000        HMI    LOS_magnetic_field 6173.0 .. 6174.0
+    2011-01-01 00:09:25.000        HMI          LOS_velocity 6173.0 .. 6174.0
+    Length = 42 rows
+    <BLANKLINE>
+    <BLANKLINE>
 
-Normal slicing operations work as with any other Python sequence, e.g.
-``results[1,::10]`` to access every tenth file in the result returned by
-the second client.
+To give an example of filtering post-search, let's only return the rows in the table which are line of sight magnetograms from or the 94Å passband from AIA.
```

Comment:
Å?

Reply:
Just because that's the units in the table so hopefully makes it a little easier to comprehend.